2021-preseason/src/main/java/frc/robot/Robot.java

```diff
@@ -25,6 +28,12 @@
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
 
   private Drive drive;
+  private Climb climb;
+  private XboxController controller;
+  private JoystickButton start; //not sure if these 4 buttons are needed cause in the API, there are methods to check fot the value of the buttons in the xboxcontroller Class/ Not used rn but keeping it here just in case
+  private Joystickbutton back;
+  private JoystickButton A;
+  private JoystickButton B;
```

Comment:
I think you don't need to create objects for all the buttons since you can get that information from the `XboxController`.